Sources/ConsoleKit/Command/Argument.swift

```diff
@@ -61,9 +64,10 @@ public final class Argument<Value>: AnyArgument
     ///
     /// - Parameters:
     ///   - help: The arguments's help text when `--help` is passed in.
-    public init(name: String, help: String = "") {
+    public init(name: String, help: String = "", completion: CompletionAction = .default) {
```

Comment:
Add documentation for the `completion` parameter. Also, it looks like we are missing documentation for the `name` parameter also.